tests/specs/v2/customTiming.js.orig

```diff
@@ -0,0 +1,61 @@
+var webdriverio = require('webdriverio');
+
+describe("Custom Timing tests", function() {
+
+  afterEach(function() {
+    /**
+     * Clears the session between tests to ensure
+     * that the sessionstart event is always fired 
+     */
+    browser.execute(function() {
+      localStorage.clear();
+    });
+  });
+
+  describe('legacy custom timings', function() {
+    beforeEach(function() {
+      browser.url('http://localhost:4567/fixtures/v2/legacyCustomTiming.html');
+      browser.pause(1000);
+    });
+
+    it('sends custom timing events', function () {
+      var customTimingData = browser.execute(function () {
+        return window.__requestPayloads[1];
+      }).value;
+
+      expect(JSON.parse(customTimingData.eventData[0].data)[0].customTiming).toEqual({
+        custom1: 100,
+        custom2: 50
+      });
+    });
+  });
+  
+  describe('new custom timings', function() {
+    beforeEach(function() {
+      browser.url('http://localhost:4567/fixtures/v2/customTiming.html');
+      browser.pause(1000);
+    });
+
+<<<<<<< HEAD
+  it('sends custom timing events', function () {
+    var customTimingData = browser.execute(function () {
+      return window.__requestPayloads[1];
+    });
+=======
+    it('sends custom timing events', function () {
+      var customTimingData = browser.execute(function () {
+        return window.__requestPayloads[1];
+      }).value;
+>>>>>>> Update custom timing tests
+
+      expect(JSON.parse(customTimingData.eventData[0].data)[0]).toEqual({
+        timing: {
+          a: "0.00",
+          du: "100.00",
+          t: "t"
+        }, 
+        url: "timingName"
+      });
+    });
+  });
+});
```

Comment:
Left merge conflict marker?

Reply:
Whoops, will remove this before merging in 👍